cmd/agent/main.go

```diff
@@ -17,65 +19,75 @@ type Config struct {
 }
 
 func main() {
+	localLogger, err := zap.NewDevelopment()
+	if err != nil {
+		panic(err)
+	}
+	defer localLogger.Sync()
+
+	logger.Sugar = *localLogger.Sugar()
+
 	var cfg Config
 	flag.StringVar(&cfg.Host, "a", "localhost:8080", "host")
 	flag.Int64Var(&cfg.ReportIntervalSeconds, "r", 10, "report interval")
 	flag.Int64Var(&cfg.PollIntervalSeconds, "p", 2, "poll interval")
 	flag.Parse()
 
-	err := env.Parse(&cfg)
+	err = env.Parse(&cfg)
 	if err != nil {
-		log.Fatal(err)
+		logger.Sugar.Fatal(err)
 	}
-	log.Println(cfg)
+	logger.Sugar.Info(cfg)
 
 	run(cfg)
 }
 
 func run(cfg Config) {
+
 	pollInterval := time.Duration(cfg.PollIntervalSeconds) * time.Second
 	reportInterval := int64(time.Duration(cfg.ReportIntervalSeconds) * time.Second / pollInterval)
-	log.Info(pollInterval)
-	log.Info(reportInterval)
+	logger.Sugar.Info(pollInterval)
```

Comment:
Можно добавить название логируемого параметра вот таким образом.
```suggestion
	logger.Sugar.Info(zap.Int64("pollInterval", cfg.PollIntervalSeconds))
```